0004-market-integration-token-standard.md

```diff
@@ -107,24 +127,24 @@ At time of writing, this standard is established with several constraints found
   }
 
   //*** Researching the following methods ***//
-  function lockToken(tokenId:string):void {
+  function lockToken(tokenId:u64):void {
     unimplemented();
   }
-  
-  function unlockToken(tokenId:string):void {
+
+  function unlockToken(tokenId:u64):void {
     unimplemented();
   }
 
   // The concept of escrow that the standard knows about is optimizing for multiple NFT exchanges
-  export function checkEscrow(escrowId:string, tokenIds:string[]) {
+  export function checkEscrow(escrowId:string, tokenIds:u64[]) {
     unimplemented();
   }
 
-  export function approveEscrow(escrowId:string, tokenIds:string[]):void {
+  export function approveEscrow(escrowId:string, tokenIds:u64[]):void {
     unimplemented();
   }
 
-  export function cancelEscrow(escrowId:string, tokenIds:string[]):void {
+  export function cancelEscrow(escrowId:string, tokenIds:u64[]):void {
```

Comment:
To save the git diffs in the future, would it be worth having a `tokenId` type (merely an alias to `u64` at present) so it is simpler to edit them all?

Reply:
Good idea, integrating this for all of these.